CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ This CHANGELOG follows the format located [here](https://github.com/sensu-plugin
 - Filter `statements` property renamed to `expressions` (@webframp)
 - Entity `class` property renamed to `entity_class`
 
+## [0.0.4] - 2019-06-19
```

Comment:
In general the versioning and dating are left to the maintainers for several reasons:
- we may disagree on the impact of the change and therefore how its versioned (generally this would be a minor but right now to help indicate its lack of maturity we are keeping to patch releases for a bit longer though this will probably change shortly).
- we date the release rather then when the contribution was submitted, PR review and acceptance has no SLA and is not guaranteed to be merged within the same day
- PRs are not a FIFO queue, as such versioning them prior to acceptance leads to needing to ask people to rebase more often

Reply:
I understand that as it relates to the status of this project, however the PR does request an update to the changelog, so it was made. This can be removed or edited if desired.

Reply:
https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#unreleased-change

I agree we can probably make this clearer but contributions should be submitted under `## Unreleased` and the maintainer will version and date it prior to release post acceptance.